assets/scss/6-components/table/_table.scss

```diff
@@ -1,23 +1,68 @@
+@mixin fixed-table-widths($widths) {
+  @for $i from 1 through length($widths) {
+    $curr: nth($widths, $i);
+
+    thead tr:first-child > th:nth-child(#{$i}),
+    tbody tr:first-child > td:nth-child(#{$i}) {
+      width: $curr;
+    }
+  }
+}
+
 // Table (c-table)
 //
-// Helper for table display
+// Tabular display classes
 //
 // .c-table--striped - Zebra stripes odd rows
+// .c-table--bordered - Bordered rows
+// .c-table--10-45-45 - Three-column table with col widths 10%, 45%, 45%
+// .c-table--15-70-15 - Two-column table with col widths 15%, 70%, 15%
 //
 // Markup: 6-components/table/table.html
 //
 // Styleguide 6.1.5
 .c-table {
+  // @extend .font-sansserif; // use font helper
   border-collapse: collapse;
+  width: 100%; // use helper class
+
+  td,
+  th {
+    // not a standard size
+    padding: .5rem;
+  }
+
+  &--10-45-45 {
+    @include fixed-table-widths(10% 45% 45%);
+    table-layout: fixed;
+  }
+
+  &--30-70 {
+    @include fixed-table-widths(30% 70%);
+    table-layout: fixed;
+  }
+
+  &--15-70-15 {
+    @include fixed-table-widths(15% 70% 15%);
+    table-layout: fixed;
+  }
 
   &--striped {
-    tr:nth-child(2n) {
-      background-color: $color-white-off;
+    tbody > tr:nth-child(odd) {
+      // not a standard color
+      background-color: #f7f7f7;
     }
   }
 
-  td,
-  th {
-    padding: $size-s;
+  &--bordered {
+    tbody > tr {
+      border-top: 1px solid $color-white-off;
+    }
   }
 }
+
+// add to td with text that should be
+// allowed to wrap mid-word
+.is-verbose {
```

Comment:
is-verbose should maybe move to our utilities folder. Also I'm not sure that I would think `is-verbose` means break-all. I'm sure I'm missing something there, but why do we want to break-all?

Reply:
Moved to typography directory as `t-break-word`.